cdap-ui/app/cdap/components/Overview/Tabs/SchemaTab/index.js

```diff
@@ -37,9 +36,14 @@ export default class SchemaTab extends Component {
 
   componentWillMount() {
     if (!this.props.entity.schema) { return; }
-
-    let fields = getParsedSchema(this.props.entity.schema);
```

Comment:
Yeah. Forgot to remove the `componentWillReceiveProps` function. It will never be used in the case of schema tab. The schematab will only be mounted and unmounted. Won't receive new props while being rendered. Will remove it.